### ISSUE TITLE
The name and future of this git repo

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse-aws#11]
@telnoratti @joelmarkanderson hi! Thanks again for pushing code to this repo. I never did get it working myself (never did try from the command line) and it there hasn't been an activity in a bit over a year and I'm wondering if you can fill me in on its status.

An inactive repo is totally fine but because we moved https://dataverse.harvard.edu to AWS yesterday (screenshot of the note on the homepage below) I'm wondering how you feel about me renaming this repo from `dataverse-aws` to `dataverse-aws-cloud-formation` or something to make it more specific.

There's a lot of Docker stuff going on in this repo, which is awesome.

I did a tiny bit of Docker/Kubernetes/OpenShift work over in https://github.com/IQSS/dataverse/pull/4168 and as a follow up to that I'm now mentoring a class where we hope to make some improvements: http://okrieg.github.io/EC500/PROJECTS/2018/all-projects.html#dataverse-scaling

Anyway, no rush on getting back to me on all of this. If I don't hear any objections I'll probably just rename the repo eventually.

![screen shot 2018-02-27 at 11 06 21 am](https://user-images.githubusercontent.com/21006/36740239-df4e5c84-1baf-11e8-9901-9afe56ce9971.png)

**Comment by joelmarkanderson** [comment]
Yes, it sounds like a good idea to change the repo name to reflect cloudformation somehow. Proceed. 

We are starting to actively develop on the Terraform -> Kubernetes path, and I've begun reconfiguring Docker containers to reflect current dataverse versions. I can let you know more once we have a good sense of scope.

**Comment by pdurbin** [comment]
@joelmarkanderson thanks for getting back to me so quickly. It's great to hear that VTTI is still interested in Dataverse. (I just did a search and found https://dataverse.vtti.vt.edu quite easily and if you'd like your installation to be added to the map at dataverse.org, please get in touch! You can read more about this at http://guides.dataverse.org/en/4.8.5/installation/config.html#putting-your-dataverse-installation-on-the-map-at-dataverse-org .)

From looking at https://www.terraform.io/docs/providers/aws/ it looks like Terraform supports AWS. If you don't mind me asking, are you still planning on deploying to AWS? Perhaps we could simply add a README to this repo to explain the current status. I can certainly wait until you've scoped out the project some more. I would be happy to add this project to the "Dev Efforts by the Dataverse Community" spreadsheet mentioned here: https://groups.google.com/d/msg/dataverse-community/X2diSWYll0w/ikp1TGcfBgAJ

If you're using Kubernetes you might be interested in checking out what @4tikhonov is up to over at https://github.com/Dans-labs/dataverse/blob/4.8.5-compose/conf/docker-eu/docker-compose.yml . As mentioned in the 2018-02-13 community call notes from https://groups.google.com/d/msg/dataverse-community/8cgk9j8JqKc/A6Y783V7AQAJ Slava is deploying to Kubernetes on Google Cloud. (You can read more about DataverseEU over at http://aussda.at/en/news/news/dataverseeu-reducing-language-barriers/ .)

**Comment by pdurbin** [comment]
@joelmarkanderson @telnoratti heads up that I'm planning on chatting with @aculich in a couple hours (10:30 eastern) in #dataverse on freenode and there's a good chance we'll talk a bit about Docker and Kubernetes if you'd like to join in. You'll also be able to catch up on any logs at http://irclog.iq.harvard.edu/dataverse/2018-03-09

**Comment by pdurbin** [comment]
@joelmarkanderson any news? You might want to take a look at https://github.com/IQSS/dataverse/issues/4665

**Comment by pdurbin** [comment]
@joelmarkanderson I just noticed that @poikilotherm mentioned terraform at https://github.com/IQSS/dataverse-kubernetes/issues/81

I'd love to introduce you two. 😄

**Comment by pdurbin** [comment]
@joelmarkanderson thanks for the following comment at https://github.com/IQSS/dataverse-kubernetes/issues/81#issuecomment-528579723

"We did briefly consider a stint with Terraform, but nothing much came of it. For a while we were running a fragile CloudFormation script that did not age well. We had plans to launch a kubernetes cluster within EC2 space (not provisioned or hosted service) for running the glassfish containers. At the moment we have fallen back to single instance VM until we steady our operations again."

As we discussed in IRC, the Dataverse team is finally getting into AWS. Harvard Dataverse is on AWS (it looks like I opened this issue the day after that switch 😄 ). Our demo site is on AWS. Developers like me spin up EC2 instances all the time for testing using a script we wrote. There's a version of the script linked from http://guides.dataverse.org/en/4.16/developers/deployment.html#deploying-dataverse-to-amazon-web-services-aws and an even newer version linked from the README at https://github.com/IQSS/dataverse-sample-data

Mostly I'm writing all this so that anyone who stumbles upon this repo can read through this issue and get a sense of where we are as a community with AWS these days. Thanks again for the contributions! Oh and these days your installation is on both the map at https://dataverse.org and https://dataverse.org/metrics 😄
